Makefile

```diff
@@ -41,7 +41,7 @@ create_new_role:
 
 .PHONY: role_molecule
 role_molecule: ## Regenerate the molecule jobs configuration
-	bash scripts/create_role_molecule
+	python3 scripts/create_role_molecule.py
```

Comment:
I'd prefer to skip this by making the script executable, just add the proper shebang `#!/usr/bin/env python3
` and add exec permissions to the file. With that you can avoid changing this line.